packages/cli/src/commands/schema/authorize.ts

```diff
@@ -4,6 +4,8 @@ import { addPermission } from '../../lib/aws-utils'
 import { lambdaAuthFlags } from '../../lib/common-flags'
 
 
+export const SCHEMA_AWS_PRINCIPAL = '148790070172'
```

Comment:
I don't personally use this format for constants in JS and I don't see it much either. The distinction used for it in the Java world just doesn't make as much sense in the JS world. Google's Java code style doc uses this format for "constants" which they then go on to define as "static final fields", which exist in JS but are rare.

https://google.github.io/styleguide/javaguide.html#s5.2.4-constant-names

Reply:
I think it is a good visual distinction for "hard coded" constants which are intended to remain unchanged, rather than just another `const` which really only can't be reassigned, but could change due to side effects, etc. It is also called out in [Mozilla's docs](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/const#basic_const_usage).